src/main/java/com/zendesk/maxwell/replication/BinlogConnectorEvent.java

```diff
@@ -62,7 +62,7 @@ public EventType getType() {
 
 	public void setLastHeartbeat(Long lastHeartbeat) {
 		this.position     = new BinlogPosition(gtidSetStr, gtid, this.position.getOffset(), this.position.getFile(), lastHeartbeat);
-		this.nextPosition = new BinlogPosition(gtidSetStr, gtid, this.position.getOffset(), this.position.getFile(), lastHeartbeat);
+		this.nextPosition = new BinlogPosition(gtidSetStr, gtid, this.nextPosition.getOffset(), this.nextPosition.getFile(), lastHeartbeat);
```

Comment:
I think this was a mistake to set current position to `nextPosition`. OpenReplicator is using nextPosition for position store.